scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestController_SpecificSchedule_Test.java

```diff
@@ -105,6 +115,50 @@ public void testCreateAndGetSchedules() throws Exception {
 
 		assertCreateAndGetSchedules(multipleAppIds, 1);
 		assertCreateAndGetSchedules(multipleAppIds, 5);
+
+	}
+
+	@Test
+	@Transactional
+	public void testCreateSchedulesMultipleConcurrent() throws Exception {
+		ExecutorService service = Executors.newFixedThreadPool(20);
```

Comment:
As long as it fails reliably, no.
I just find it a bit odd that we are using reflection to decide what methods to invoke. It would make more sense if we just enumerate the calls.